src/compiler/diagnosticMessages.json

```diff
@@ -983,6 +983,18 @@
         "category": "Error",
         "code": 1343
     },
+    "This parameter is not allowed with 'use strict' directive.": {
+        "category": "Error",
+        "code": 1344
+    },
+    "'use strict' directive cannot be used with non simple parameter list.": {
+        "category": "Error",
+        "code": 1345
+    },
+    "{0} is here.": {

```

Comment:
I imagine this is a localization hazard. `parameter declared here` should be written in full, instead of a "generic" message, so "parameter" gets localized.